app/views/registration_mailer/registration_email.text.erb

```diff
@@ -0,0 +1 @@
+<%= strip_tags @body %>
```

Comment:
I think using `strip_tags` here means that the link with the token won't be visible in the output email, so the text version won't be usable. It might be worthwhile including only the common text in the body and include differently-formatted links in the HTML and text version?

Reply:
Yes indeed. At the very least those links need to be readable as opposed to `<a ...>click here</a>` formatting.

I think I'd rather see the whole e-mail previewed and editable in the preview page, otherwise there's hidden stuff going into the outgoing mails that is either invisible to the editor or displayed around the ckeditor pane.

Let's see what I can do with the source HTML template. I haven't dealt with previewing Mailer output since we first worked on the approval requests, but I need to add to `test/mailers/previews` for starters. Then I can get a better idea about what needs to be done.

Reply:
Good point regarding seeing the entire email in the preview.

Reply:
I've tried to push the parameters inside the mailer towards more "leaf node" than primitive (specify finalize URL rather than deriving it from the registration, the controller, and God knows what else inside the mailer). This approach seems more preview-friendly since Rails' Action Mailer preview functionality seems to be a duct-taped collection of hacks.

Reply:
The text preview appears to be acceptably equivalent to the HTML.
http://localhost:3000/useradmin/rails/mailers/registration_mailer/registration_email
... since I can never remember the URL